ifStatement.js

```diff
@@ -13,7 +13,13 @@ Note:
    You can change the value of the age variable to test different cases. If the person's age is less than the minimum driving age, nothing will be printed.
 ************************************************************************************************/
 // TODO: ADD YOUR CODE BELOW
-
+let userAge = 23;
+let  minimum=18;
```

Comment:
Well done!
It's better to use a clear variable name. Here, the name "minimum" doesn't clearly indicate what it refers to. Consider using a more descriptive name such as "minDrivingAge".